symfony/framework-bundle/6.4/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.yml]
+indent_size = 2
```

Comment:
FYI, I chose 2 here because of github actions. I find spacing in these files all over the place but most official sources use 2 spaces.

Reply:
We are using 4 spaces (consistently) in Symfony :)

Reply:
I've removed but note that even the [Symfony uses 2 spaces in their GHA .yml files](https://github.com/symfony/symfony/blob/7.3/.github/workflows/package-tests.yml).

Maybe?

```
[.github/**.yml]
indent_size = 2
```

Reply:
All Symfony configuration files (everything in this repo) are using 4 spaces.
GHA files are outside of the scope here.